### PR TITLE
PSDK-108 GREENPLUM support

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/base.py [hunk 1]
```diff
@@ -9,8 +9,9 @@
 T = TypeVar("T", bound="Base")
 
 
-def value_in_allowed(instance: Type[Base], attribute: attr.Attribute, value: str) -> None:
-    client_class = instance.client_class()
+def value_in_allowed(instance: Type[Base], attribute: attr.Attribute, value: str, client_class: Any = None) -> None:
```
Comment (pcerny) [inline gooddata-sdk/gooddata_sdk/catalog/base.py:12]: `client_class` can be `None`. Reflect that in type annotation.

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR167

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/3/ : SUCCESS in 1m 33s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/153/ : SUCCESS in 1m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/155/ : SUCCESS in 22s

**Pull request merged**